src/main/resources/fabric.mod.json

```diff
@@ -1,6 +1,6 @@
 {
   "schemaVersion": 1,
-  "id": "${id}",
+  "id": "autoreconnect",
```

Comment:
Shouldn't the mod id be set automatically in the build process? Maybe change that back, too.